ei.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
This looks good. As you mentioned, we can document this spec later in code and use it to validate the input files.